### PR TITLE
Task#7 - WEB TCP/IP

### PR DESCRIPTION
**Pull request by Jagrmi-C**

### CODE REVIEW

File: courses/task_eight_GP_server/main.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"log"
+    "net/http"
+    "github.com/Jagrmi-C/gostarted/courses/task_eight_GP_server/handlers"
+)
+
+func main() {
+    router := mux.NewRouter()
+    router.HandleFunc("/api/v0/", handlers.GETRootHandler).Methods(http.MethodGet)
+    router.HandleFunc("/api/v0/", handlers.POSTRootHandler).Methods(http.MethodPost)
+    router.HandleFunc("/api/v0/logout/", handlers.DeleteCookie).Methods(http.MethodGet)
```
Comment (wshaman) [inline courses/task_eight_GP_server/main.go]: :+1:

File: courses/task_eight_GP_server/handlers/root.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        Name:    name,
+        Value:   value,
+		Expires: expire,
+		MaxAge: 10,
+	}
+	fmt.Println("EX", expire)
+    http.SetCookie(w, &cookie)
+}
+
+func DeleteCookie(w http.ResponseWriter, r *http.Request) {
```
Comment (wshaman) [inline courses/task_eight_GP_server/handlers/root.go]: Name messing. Reading these names one will think `DeleteCookie` as opposite method for `addCookie`.

File: courses/lib/errorCustom.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package lib
+
+import (
+	"log"
+	"net/http"
+)
+
+func ReturnInternalError(w http.ResponseWriter) {
```
Comment (wshaman) [inline courses/lib/errorCustom.go:8]: :-D

**Review by wshaman — APPROVED** [review]

File: courses/lib/errorCustom.go (hunk 3 above)
Comment (Jagrmi-C) [reply courses/lib/errorCustom.go:8]: copypasta)))))

**Pull request merged**